### PR TITLE
Update FoJS post to clarify how stable releases work

### PR DESCRIPTION
**Pull request by jdesrosiers**
This attempts to improve the wording in the last bullet point based on feedback from @karenetheridge.

### CODE REVIEW

File: pages/posts/future-of-json-schema.md [hunk 1]
```diff
@@ -122,9 +122,9 @@ finalized, but here are some outcomes users can expect.
   you won't have to continue to update them as JSON Schema evolves.
 * Custom dialects and vocabularies will continue to be central concepts for
   customizing and extending JSON Schema.
-* Implementations only need to support the latest stable release. For example, a
-  library that supports the 2025 release will automatically support the 2023 and
-  2024 releases. The older releases will no longer need to be maintained as
-  distinct versions. However, implementations that continue to support the
+* Implementers won't need to maintain code for past stable releases. A library
```
Comment (handrews) [inline pages/posts/future-of-json-schema.md:125]: Perhaps "maintain distinct code"?  That's really the difference- the code you maintain for 2025 will also maintain support for 2024, 2023, etc., as opposed to switching among codebases per-draft as happens now.  And it matches "distinct" in the next part.

Comment (jdesrosiers) [reply pages/posts/future-of-json-schema.md]: I gave it another try. Let me know what you think.

**Review by handrews — APPROVED** [review]
Looks good to me, thanks!

**Review by Julian — APPROVED** [review]

**Comment by jdesrosiers** [comment]
I haven't gotten a response from @karenetheridge, but it's been over two weeks and there are two approvals, so I'm going to merge.

**Pull request merged**